packages/graphback-runtime-mongodb/src/MongoDBDataProvider.ts

```diff
@@ -165,6 +173,61 @@ export class MongoDBDataProvider<Type = any, GraphbackContext = any> implements
     return document;
   }
 
+  protected getIndexFields(baseType: GraphQLObjectType): IndexSpecification[] {
+    const res: IndexSpecification[] = [];
+    const fields = baseType.getFields();
+    Object.keys(fields).forEach((k: string) => {
+      const field = fields[k];
+
+      // Add custom Index if found e.g. @db(index: 1)
```

Comment:
This comment is not valid anymore, 

I believe it is `@index(name: ....)` now right?

Reply:
Docs docs docs